### PR TITLE
23 done - vinz

### PR DESCRIPTION
**Pull request by vinc86**

### CODE REVIEW

File: 23-PB-nested-loops/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    console.log(star);
+}
+
+
+//reverse tree
+
+// i really have no idea how i get it right :D
+
+var n = 5;
```
Comment (martl) [inline 23-PB-nested-loops/index.js:23]: please dont use var, use let

Comment (martl) [reply 23-PB-nested-loops/index.js:23]: actually, i think you can delete it

File: 23-PB-nested-loops/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+console.log();
+
+for (let i = 0; i <= 20; i++) {
+    let star = ""
```
Comment (martl) [inline 23-PB-nested-loops/index.js:27]: dont forget semicolons

File: 23-PB-nested-loops/index.js (hunk 1 above)
Comment (vinc86) [reply 23-PB-nested-loops/index.js:25]: yes, i forget to delete it, it was from a previous attempt

**Pull request closed without merging**